eng/helix/content/runtests.sh

```diff
@@ -86,7 +86,10 @@ fi
 sync
 
 exit_code=0
+echo "Restore for RunTests..."
 $DOTNET_ROOT/dotnet restore RunTests/RunTests.csproj --source https://api.nuget.org/v3/index.json --ignore-failed-sources
+echo "Running tests..."
 $DOTNET_ROOT/dotnet run --project RunTests/RunTests.csproj -- --target $1 --sdk $2 --runtime $3 --queue $4 --arch $5 --quarantined $6 --ef $7
```

Comment:
@BrennanConroy is the naked -- needed/intentional?  If so what does that do?

Reply:
Yes, it makes the rest of the arguments go to the process instead of `dotnet run`
For example, if we had an argument for --project in the RunTests project, if you didn't have -- then dotnet run would see --project and try to use it

Reply:
Got it thanks